install-calico.tf

```diff
@@ -1,18 +1,19 @@
 resource "null_resource" "calico" {
-  count = "${var.calico_enabled ? 1 : 0}"
+  count = var.calico_enabled ? 1 : 0
 
   connection {
-    host        = "${hcloud_server.master.*.ipv4_address}"
-    private_key = "${file(var.ssh_private_key)}"
+    host        = hcloud_server.master.*.ipv4_address
+    private_key = file(var.ssh_private_key)
   }
 
   provisioner "remote-exec" {
-    inline = <<EOF
-    kubectl apply -f https://docs.projectcalico.org/v3.2/getting-started/kubernetes/installation/hosted/etcd.yaml
-    kubectl apply -f https://docs.projectcalico.org/v3.2/getting-started/kubernetes/installation/rbac.yaml
-    kubectl apply -f https://docs.projectcalico.org/v3.2/getting-started/kubernetes/installation/hosted/calico.yaml
-    EOF
+    inline = [
+      "kubectl apply -f https://docs.projectcalico.org/v3.2/gettingtarted/kubernetes/installation/hosted/etcd.yaml",
```

Comment:
```suggestion
      "kubectl apply -f https://docs.projectcalico.org/v3.2/getting-started/kubernetes/installation/hosted/etcd.yaml",
```

Reply:
Don't know how this happened but now its fixed and squashed again..